src/main/java/org/folio/marc4ld/service/condition/ConditionCheckerImpl.java

```diff
@@ -44,11 +44,15 @@ public boolean isMarc2LdConditionSatisfied(Marc4BibframeRules.FieldRule fieldRul
     var ind1Condition = isSingleConditionSatisfied(String.valueOf(dataField.getIndicator1()), condition.getInd1());
     var ind2Condition = isSingleConditionSatisfied(String.valueOf(dataField.getIndicator2()), condition.getInd2());
     var fieldConditions = isEmpty(condition.getFields()) || condition.getFields().entrySet().stream()
-      .allMatch(fieldCondition -> ofNullable(dataField.getSubfield(fieldCondition.getKey()))
+      .anyMatch(fieldCondition -> ofNullable(dataField.getSubfield(fieldCondition.getKey()))
```

Comment:
This should be `allMatch` right?

Reply:
thanks, forget to return during finding solution